plugins/engine-configuration/ui/src/main/java/org/pentaho/di/engine/ui/RunConfigurationDelegate.java

```diff
@@ -59,6 +65,34 @@ public void edit( RunConfiguration runConfiguration ) {
       configurationManager.delete( key );
       configurationManager.save( savedRunConfiguration );
       spoonSupplier.get().refreshTree();
+
+      updateLoadedJobs( key, savedRunConfiguration );
+    }
+  }
+
+  protected void updateLoadedJobs( String key, RunConfiguration runConfig ) {
+    for ( JobMeta job : spoonSupplier.get().getLoadedJobs() ) {
+      for ( int i = 0; i < job.nrJobEntries(); i++ ) {
+        JobEntryInterface entry = job.getJobEntry( i ).getEntry();
+
+        if ( entry instanceof JobEntryTrans ) {
+          JobEntryTrans jet = (JobEntryTrans) entry;
+
+          if ( jet.getRunConfiguration().equals( key ) ) {
+            try {
+              ExtensionPointHandler.callExtensionPoint( job.getLogChannel(), KettleExtensionPoint.JobEntryTransSave.id,
+                new Object[] { job, runConfig.getName() } );
+            } catch ( KettleException e ) {
+              // Ignore errors
```

Comment:
(minor) Evaluate to handle the exception when it's threw (e.g. by logging the stack trace, etc...).